blog_post_collaboration/kedro_ibis.md

```diff
@@ -0,0 +1,263 @@
+│ orders        │
+│ raw_customers │
+│ raw_orders    │
+│ raw_payments  │
+│ stg_customers │
+│ stg_orders    │
+│ stg_payments  │
+└───────────────┘
+
+>>> con.sql("SELECT * FROM customers WHERE customer_id = 42")
+┌─────────────┬────────────┬───────────┬─────────────┬───────────────────┬──────────────────┬─────────────────────────┐
+│ customer_id │ first_name │ last_name │ first_order │ most_recent_order │ number_of_orders │ customer_lifetime_value │
+│    int64    │  varchar   │  varchar  │    date     │       date        │      int64       │         double          │
+├─────────────┼────────────┼───────────┼─────────────┼───────────────────┼──────────────────┼─────────────────────────┤
+│          42 │ Diana      │ S.        │ 2018-02-04  │ 2018-03-12        │                2 │                    27.0 │
+└─────────────┴────────────┴───────────┴─────────────┴───────────────────┴──────────────────┴─────────────────────────┘
+
+```
+
+As always, we can view the pipeline using Kedro-Viz:
+
+<img width="1470" alt="image" src="https://github.com/kedro-org/kedro-devrel/assets/14007150/9fedaf00-ecf6-418f-9c1e-a98844994ba2">
+
+## But wait... there's more!
+
+Making pipeline productionisation easier alone may justify adopting Ibis in your Kedro workflows, but other situations from my past experience also come to mind.
+
+Have you ever found yourself developing data pipelines against an existing data warehouse, even though you know that the data infrastructure team is currently migrating to a different database solution? You expect to rewrite some amount of your code—unless you use Ibis?
+
+Leveraging Ibis can also help you build truly reusable pipelines. I previously led development of a Kedro-based code asset that provided a suite of reusable pipelines for customer analytics. We decided to use PySpark for data engineering, but many of our users didn't need to use PySpark. One major retailer stored all of their data in Oracle, and they ended up with a suboptimal workflow wherein they extracted data into Spark and did all the work there. Other teams looked at our pipelines for inspiration, but ended up rewriting them for their infrastructure—not our intention in delivering prebuilt pipelines. Last but not least, we spent so much time and effort setting up Spark on locked-down Windows machines, so data scientists could play with the pipelines we provided; being able to run the same logic using DuckDB or pandas locally would have been a godsend!
+
+## What's next?
+
+If you're familiar with dbt (or even if you examined the Jaffle Shop project discussed above), you'll notice a key functionality that we didn't implement here: [validations](https://docs.getdbt.com/docs/build/validation). Kedro supports data validation through third-party plugins such as [kedro-pandera](https://kedro-pandera.readthedocs.io/en/latest/), and I've recently started work on extending pandera to support validating Ibis tables; look for a follow-up post covering that soon.
```

Comment:
might be worth pointing out you can also use `pytest` or other standard testing frameworks w/ Ibis